<launch>
    <arg name="puck_tracker_config" default="$(find airhockey_vision)/config/detect_blue.yaml" />
    <rosparam file="$(arg puck_tracker_config)" command="load" />

    <node pkg="airhockey_vision" type="puck_tracking_node" name="puck_tracking">
    </node>

</launch>
